config/jobs/ppc64le-cloud/build-docker/postsubmit-build-docker.yaml

```diff
@@ -135,15 +127,3 @@ postsubmits:
 
           securityContext:
             privileged: true
-          volumeMounts:
-          - name: docker-root
-            mountPath: /var/lib/docker
-          env:
-            - name: S3_SECRET_AUTH
-              valueFrom:
-                secretKeyRef:
-                  name: docker-s3-credentials
-                  key: password
-        volumes:
-        - name: docker-root
-          emptyDir: {}
```

Comment:
preset is missing in this job for this removal

Reply:
Missed adding. Will do it in the next patch.